flink-formats/flink-csv/pom.xml

```diff
@@ -58,31 +58,24 @@ under the License.
 			<scope>provided</scope>
 		</dependency>
 
-		<!-- test dependencies -->
-
-		<!-- CSV table descriptor testing -->
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-table-common</artifactId>
+			<artifactId>flink-table-api-java-bridge_${scala.binary.version}</artifactId>
```

Comment:
We cannot do this because it would introduce a Scala suffix to the module. So far we could keep Scala out of the new format module for non-testing scopes.

Reply:
However, this will avoid us to move `CsvTableSource` `CsvTableFactory` to `flink-csv` module.

Reply:
@twalthr , just as @wuchong mentioned, `CsvTableSource` and `CsvTableSink` are in `flink-table-api-java-bridge` now,  and `CsvTableSourceFactoryBase` and `CsvTableSinkFactoryBase` depend on them. one solution is : porting `CsvTableSource` and `CsvTableSink` to `flink-csv` ?

Reply:
I think having the Csv source/sink/factories in `flink-table-api-java-bridge` is the mid-term solution until we have new unified source/sink interfaces ready in `flink-table-common`. Adding a dependency to `flink-table-api-java-bridge` in `flink-csv` would require users to update their dependencies. And we decided to let `flink-formats` Scala-free in Flink 1.7.

Reply:
+1 to go into `flink-table-api-java-bridge` for now.